roles/openshift_prometheus_operator/tasks/install_prometheus_operator.yaml

```diff
@@ -13,34 +13,28 @@
         config_base: /etc/origin
  
 - name: Copy files to temp directory
-  command: >
-    cp -r /usr/share/ansible/openshift-ansible/roles/openshift_prometheus_operator/files/{{ item }} 
-    {{ mktemp.stdout }}/
-  with_items:
-  - manifests
-  - assets
-  - project.yaml
- 
+  copy:
+    src: "."
+    dest: "{{ mktemp.stdout }}/"
+    directory_mode: yes
+  
+
 - name: Copy admin client config
   command: >
     cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
   changed_when: false
  
-- name: Ensure monitoring project exists
-  command: >
-    {{ openshift_client_binary }} apply -f "{{ mktemp.stdout }}/project.yaml"
```

Comment:
if the role is not using `project.yaml` you should probably delete that file.

Reply:
Nice catch. Thanks